template/_gitignore

```diff
@@ -71,4 +71,7 @@ dist/
 #Electron cache
 .electron/
 .webpack/
-out/
\ No newline at end of file
+out/
+
+#FPE plugin folder TODO:hari rename folder later
+template/test-fpe/*
```

Comment:
NOTE: Remove the TODOs and move it to a separate file with the pending items.

Reply:
Done